tests/test_sample.py

```diff
@@ -134,11 +134,11 @@ def test_paulix_pauliy(self, theta, phi, varphi, device, shots, tol):
         dev.apply("CNOT", wires=[1, 2], par=[])
 
         dev._obs_queue = [
-            Tensor(["PauliX", "PauliY"], [[0], [2]], [[], []], qml.operation.Sample)
+            qml.PauliX(0, do_queue=False) @ qml.PauliY(2, do_queue=False)
         ]
 
-        # for idx in range(len(dev._obs_queue)):
-        #     dev._obs_queue[idx].return_type = qml.operation.Sample
+        for idx in range(len(dev._obs_queue)):
+            dev._obs_queue[idx].return_type = qml.operation.Sample
```

Comment:
These lines do not seem to affect the outcome of the test, is there a particular reason to include them?

Reply:
Yep - since we are not using the high-level `sample()` function, the sample return type on the observables is not being set. Without this being set, the plugin will not instruct Qiskit to return the samples (this is optional in qiskit).